Description: Debianizing
 TODO: Put a short summary on the line above and replace this paragraph
 with a longer explanation of this change. Complete the meta-information
 with other relevant fields (see below for details). To make it easier, the
 information below has been extracted from the changelog. Adjust it or drop
 it.
 .
 selenium (2.21.2-1) unstable; urgency=low
 .
   * source package automatically created by stdeb 0.6.0+git
Author: Andrew King (No comment) <aking1012.com@gmail.com>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- selenium-2.21.2.orig/MANIFEST.in
+++ selenium-2.21.2/MANIFEST.in
@@ -14,7 +14,6 @@ recursive-include py/selenium/webdriver/
 recursive-include py/selenium/webdriver/support *.py
 include py/selenium/selenium.py
 include py/selenium/__init__.py
-include docs/api/py/index.rst
 include py/CHANGES
 include py/README
 recursive-include selenium.egg-info *
